src/jit/morph.cpp

```diff
@@ -3844,6 +3843,9 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* call)
             else
             {
                 // This is passed by value.
+                CLANG_FORMAT_COMMENT_ANCHOR;
+
+#ifndef _TARGET_X86_
                 // Check to see if we can transform this into load of a primitive type.
```

Comment:
Looks like we do not have any code for x86 under `// This is passed by value.` 
Do we pass structs by value on x86?

Reply:
Yes, we pass structs by value on x86, but we don't do the transformation to a primitive type, so we decide to copy based only on the criteria below (in the code that was accidentally ifdef'd out).